testbed/eks/lib/addons/fluxv2.ts

```diff
@@ -0,0 +1,110 @@
+import cdk = require('@aws-cdk/core');
```

Comment:
let's avoid including versioning concepts as part of files names (e.g. fluxv2.ts -> flux.ts)

Reply:
Also -- consider creating one package per addon, since we might end up having many files in support of the addon. Off the top of my head, we'll want to separate cloud formation templates and typescript code. If all addons are in the same package, this is going to get unwieldy.

Reply:
huge +1 on this. We should be modeling all of these as layers and these layers should be easily swapped as the add-ons might change, config might change, etc.

Reply:
Also, do we need the `lib` directory under eks? Can it be just `eks/addons`?